reactive/k8s.py

```diff
@@ -346,6 +359,14 @@ def get_dns_ip(cidr):
     return '.'.join(ip.split('.')[0:-1]) + '.10'
 
 
+def get_sdn_ip(cidr):
```

Comment:
Mark my words, these will bite us eventually, but until then we shall dance madly on the lip of an active volcano.